sdcm/utils/sstable/load_utils.py

```diff
@@ -60,30 +60,29 @@ def upload_sstables(node, test_data: TestDataInventory, keyspace_name: str = 'ke
                         hash_expected=test_data.sstable_md5, retries=2,
                         user_agent=creds['user_agent'])
 
-        with RemoteTemporaryFolder(node=node) as tmp_folder:
-
-            if node.is_docker():
+        if create_schema:
```

Comment:
so now you untar the file twice ? in case of create schema ?

Reply:
is so please comment the reason you are doing it twice, so no one would try to "optimize" it again.

Reply:
added coments